src/MahApps.Metro/MahApps.Metro.Shared/Controls/Helper/TextBoxHelper.cs

```diff
@@ -31,6 +31,8 @@ public class SpellCheckSeparator : Separator, ISpellCheckMenuItem
     {
     }
 
+    public enum FloatingWatermarkLocations { Interior, Exterior };
```

Comment:
Could you consider renaming this enum to singular. Because plural enum name should be used for enums with a [FlagAttribute]